airflow/providers/cncf/kubernetes/utils/pod_launcher.py

```diff
@@ -51,10 +54,26 @@ def should_retry_start_pod(exception: Exception) -> bool:
 class PodStatus:
```

Comment:
any reason not to go ahead and rename this to `PodPhase` while we're at it?

Reply:
+1 for `PodPhase`!